### DISCUSSION TITLE
Please help 😣 Idk what I did wrong.

### DISCUSSION THREAD

**Discussion by katastrophe92** [espanso/espanso#1869] [Q&A] [answered]
Hello espanso team and community! Please let me know what is the problem. I had all of these triggers in a single .yaml file and today I ordered to different.yml files. Besides copy the text from one file to other file, I did nothing. Hope some will understand what I mean and help me. Thanks!

**Comment by smeech** [comment, marked as answer]
Duplicate of https://github.com/espanso/espanso/issues/1879 and probably answered in https://discord.com/channels/884163483409731584/1013918746471039067/1212886163417866301
